docs/howto/getting_started_markdown.md

```diff
@@ -0,0 +1,173 @@
+Getting Started Guide
+=====================
+
+Prerequisites
+-------------
+
+To get started with the Alteryx Core SDK, you need a valid installation
+of [Alteryx Designer](https://www.alteryx.com/).
+
+Python version 3.8.5 is required for Alteryx Designer plugins.
+
+Set Up the Development Environment
+----------------------------------
+
+Follow these steps to configure your development environment.
+
+### Create the Virtual Environment
+
+To get started\...
+
+1.  Install [Miniconda3](https://docs.conda.io/en/latest/miniconda.html)
+    for your system.
+2.  Once the install is done, open an Anaconda Prompt and create a new
+    virtual environment: `conda create -n ayx_python_sdk python=3.8.5`.
+3.  Next, activate it: `conda activate ayx_python_sdk`.
+
+If you are not familiar with Anaconda/Miniconda, visit [Anaconda
+Documentation](https://docs.anaconda.com/anaconda/user-guide/getting-started/).
+
+### Install the Package
+
+After you create and activate your virtual environment, pip install the
+Alteryx Core SDK pip package. For more information on pip, visit the
+[pip website](https://pypi.org/).
+
+Use this command to install the SDK:
+
+`pip install ayx-python-sdk`
+
+If you do not already have the Alteryx Plugin CLI, use this command to
+install the CLI:
+
+`pip install ayx-plugin-cli`
+
+### Create the Ayx Plugin Workspace
+
+Now that the Alteryx Python SDK is installed in your virtual environment
+packages, we\'ll create a new [Plugin Tool
+Workspace](https://extensibility.pages.git.alteryx.com/ayx-sdks/getting_started.html#glossary)
+with default files ready to go.
+
+Use this command to create the workspace:
+`ayx_plugin_cli sdk-workspace-init`
+
+This initializes a workspace to manage all of your custom tools and
+plugins.
+
+To create a new tool, use the `create-ayx-plugin` command.
+
+The `create-ayx-plugin` command takes these parameters:
+
+ -   `--tool-name`: (Required) The name of the new Plugin tool
+ -   `--tool-type`: The type of tool to create.
+
+For this example we\'ll name our new tool `MyFirstTool`. We\'ll use the
+tool-type `single-input-single-output` parameter to create the new tool
+with default input and output anchors.
```

Comment:
TW review:

For this example, name your new tool `MyFirstTool`. Use the
tool-type `single-input-single-output` parameter to create the new tool
with default input and output anchors.